src/components/Statistic/Statistic.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+class Statistic extends React.Component {
+
+    constructor(props) {
```

Comment:
As we aren't doing anything except `super` I don't think we need a constructor here

Reply:
on it